### PR TITLE
S18 kindergarden and group models

### PR DESCRIPTION
**Pull request by Riaboshtan**
added models of Kindergarten and Group

### CODE REVIEW

File: apps/kindergarten/models.py [hunk 1]
```diff
@@ -1 +1,30 @@
...
+from core.models.helpers import image_path
+
+
+class Kindergarden(models.Model):
+    name=models.CharField(max_length=200)
+    image=models.ImageField(
+        upload_to=image_path,
+        blank=True
+    )
+    disctict=models.CharField(max_length=50)
```
Comment (aodarc) [inline apps/kindergarten/models.py:13]: in the future we should create FK to location.City and location.District

Comment (aodarc) [reply apps/kindergarten/models.py:13]: word mistake is found)

File: apps/kindergarten/admin.py [hunk 2]
```diff
@@ -1 +1,7 @@
 # Register your models here.
+from django.contrib import admin
+
+from apps.kindergarten.models import Kindergarden, Group
```
Comment (aodarc) [inline apps/kindergarten/admin.py]: from apps.kindergarten.models import (
Group,    
Kindergarten
)

Comment (aodarc) [reply apps/kindergarten/admin.py]: word mistake must be **Kindergarten**

File: apps/kindergarten/admin.py [hunk 3]
```diff
@@ -1 +1,7 @@
...
+
+admin.site.register(Kindergarden)
```
Comment (aodarc) [inline apps/kindergarten/admin.py]: word mistake must be **Kindergarten**

File: apps/kindergarten/models.py [hunk 4]
```diff
@@ -1 +1,30 @@
 # Create your models here.
+from django.db import models
+
+from core.models.helpers import image_path
+
+
+class Kindergarden(models.Model):
```
Comment (aodarc) [inline apps/kindergarten/models.py]: word mistake must be **Kindergarten**

**Review by aodarc — CHANGES_REQUESTED** [review]
correct mistakes by following my comments.
To rename Model you should delete migration of that model from */migrations/* folder
also from "django_migrations" delete those migration
and tables

File: apps/kindergarten/models.py (hunk 1 above)
Comment (Riaboshtan) [reply apps/kindergarten/models.py:13]: I fixed mistakes, but I don't understand what to do with "in the future we should create FK to location.City and location.District" ?

**Review by aodarc — COMMENTED** [review]
ready to merge

**Review by aodarc — APPROVED** [review]

**Comment by maziar0andriy** [comment]
fine

**Pull request merged**